src/config/ai-providers.ts

```
@@ -0,0 +1,223 @@
+      if (!openaiConfig.apiKey) {
+        return null;
+      }
+      
+      return openaiConfig;
+
+    case 'gemini':
+      const geminiConfig = {
+        provider: 'gemini' as const,
+        apiKey: import.meta.env.VITE_GEMINI_API_KEY,
+        baseUrl: import.meta.env.VITE_GEMINI_BASE_URL || 'https://generativelanguage.googleapis.com/v1beta',
+      };
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)